youtube_dl/extractor/piksel.py

```diff
@@ -113,11 +101,25 @@ def _real_extract(self, url):
             })
         self._sort_formats(formats)
 
+        subtitles = {}
+        for caption in video_data.get('captions', []):
+            caption_url = caption.get('url')
+            caption_format = caption.get('url').split(' ')[-1].split('.')[-1]
```

Comment:
this can break the extraction, and those checks look unnecessary, it's possible that other videos will have other types of subtitles(srt,ttml,scc...).

Reply:
agreed. will address as suggested.